moj-be/modules/custom/moj_healthcheck/src/Controller/HealthcheckApiController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Drupal\moj_healthcheck\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Database;
+
+class HealthcheckApiController extends ControllerBase
+{
+    public $response;
+
+    public function healthcheckApiEndpoint()
+    {
+        $this->response = $this->setResponse();
+        return $this->response;
```

Comment:
I'd be nice to make this a bit richer - include the application version number / git reference etc. Not essential though.

It should also return a 200 if OK and a 500 if not.

Reply:
The git thing is going to take a bit more work which I'll card up.. 
If the database is down then drupal will give a 500